windows-installer/Package.wxs

```diff
@@ -10,33 +10,32 @@
 		<Feature Id="main">
 			<!-- Include files that has to be installed -->
 			<Files Include="msi-files\**" />
+      <Component Id="setEnv" Directory="INSTALLFOLDER" Guid="8bea8575-87ca-4847-8b8b-202b6e34901e">
+        <CreateFolder/>
+        <Environment Id="IdeRoot" Name="IDE_ROOT" Value="[IDE_ROOT]" Permanent="yes" Action="set" System="no" />
+      </Component>
 		</Feature>
 
 		<MediaTemplate EmbedCab="yes" />
-
-		<!-- Specifying license and bitmap files -->
 		<WixVariable Id="WixUILicenseRtf" Value=".\assets\LICENSE.rtf" />
 		<WixVariable Id="WixUIBannerBmp" Value=".\assets\banner.bmp" />
 
-		<!-- Specifying Directories and Properties for installation location selection-->
-		<Property Id="INSTALLTYPE" Value="ROOT" />
-		
-		<Directory Id="ROOTDIRECTORY">
-			<Directory Id="INSTALLFOLDER" Name="projects\_ide"/>
-		</Directory>
-		
-		<Property Id="USERFOLDER" >
+		<Property Id="INSTALLTYPE" Value="USERHOME" />
+	  <Property Id="USERFOLDER" >
 			<DirectorySearch Id="userProfileSearch" Depth="0" Path="[%USERPROFILE]" />
 		</Property>
-
 		<Directory Id="USERFOLDER" >
-			<Directory Id="USERHOMEFOLDER" Name="projects\_ide" />
+			<Directory Id="IDE_ROOT" Name="projects" />
 		</Directory>
+		<Directory Id="ROOTDIRECTORY" />
+
+    <CustomAction Id="SetIDE_ROOT" Property="IDE_ROOT" Value="[ROOTDIRECTORY]projects" Execute="immediate" Return="check" />
+    <CustomAction Id="SetTEMP_FOLDER" Property="INSTALLFOLDER" Value="[IDE_ROOT]\_ide\temp\ideasy" Execute="immediate" Return="check" />
 
 		<!-- Execution of install command-->
 		<SetProperty
 			Id="RunInstallAction"
-			Value="&quot;[%SystemFolder]cmd.exe&quot; /c &quot;[INSTALLFOLDER]setup.bat -b&quot;"
+			Value="&quot;[%SystemFolder]cmd.exe&quot; /c &quot;cd /d &quot;[INSTALLFOLDER]&quot; &amp;&amp; bin\ideasy.exe -f install&quot;"
```

Comment:
Found a way to run `bin\ideasy.exe -f install`

Reply:
Excellent. Great that you made it work.